src/components/tables/PoolsTable/PoolsTable.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <BalCard shadow="lg" class="mt-4" no-pad>
+  <BalCard shadow="lg" class="mt-4" no-pad square>
```

Comment:
This should be dependent on the breakpoint. I don't think the table should be max width and square on screen sizes above mobile. For example on my 15" macbook there should be padding around the tables as it was before.

This is what it looks like full screen for me now.
<img width="1440" alt="Screenshot 2021-07-13 at 14 42 13" src="https://user-images.githubusercontent.com/2406506/125462755-0237b7c0-848d-4c95-abef-5a1b50d43917.png">

Reply:
This was intentional but I think we can do better. Basically, if the table can be scrolled horizontally, it definitely needs to extend to the right hand side edge of the screen. Otherwise, people may not realize that there is additional content, and scrolling horizontally within a container feels weird. 

I'll try adding left padding to the tables, but I definitely want to extend the table to the right hand edge until it reaches it's full width.

Reply:
I don't think we're on the same page here. On my laptop the table extends the full width of the page when there is no need. Currently on production it's like this for me (full screen):

<img width="1440" alt="Screenshot 2021-07-13 at 19 42 43" src="https://user-images.githubusercontent.com/2406506/125507641-97d24d09-dbf1-4f5b-89ec-5919bb65c684.png">

Which is correct, the table should only go full-width when the screen size prevents the whole table from being seen. We just need to use the breakpoints correctly here. I can help with this.

Reply:
I've done another version which I think works better. The tables start full width of mobile, then add left padding on small screens upwards. Once it hits `lg`, I place padding on both sides. Feels pretty fluid now.

Reply:
Hey, so Tailwind gives a suggestion of how to use the container class with a breakpoint so it feels fluid [here](https://tailwindcss.com/docs/container#responsi). I've updated to use that in this [commit](https://github.com/balancer-labs/frontend-v2/pull/558/commits/395c3dea2e399bec879d7d955ef247b71d8c0d81).

I also don't think we need to go to absolute full width and square for the tables on mobile. I think we can maintain the border radius and just have a much smaller padding. I've included that in the above commit too, let me know what you think. I feel like if we want to have absolute full width for tables on mobile we need to remove border radius, border colors, and shadows and actually design a full screen experience that feels more like a mobile app.